front/src/components/surveyExport.js

```diff
@@ -0,0 +1,118 @@
+import axios from 'axios'
+import React, {Component} from "react";
+import {Button} from "reactstrap";
+import {Link} from "react-router-dom";
+
+class SurveyExport extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            survey: -1,
```

Comment:
This integer value might be confusing - maybe an empty string `""` would be better?